pkg/lspext/lspext.go

```diff
@@ -0,0 +1,35 @@
+package lspext
```

Comment:
In the Sourcegraph repo, we call this `lspx` not `lspext`. I don't care which one we use, but it should be consistent.